### ISSUE TITLE
TL 90 Raygun Cannon low dmg

### ISSUE THREAD

**Issue by NCKryptonite** [NCC-Lykos/neocron-classic#239]
Currently at 175% dmg (564 total enr+xrr). Direct hit dmg is maxing at 261 on a 50/50.

Whereas the TL 78 Raygun cannon I have 178% dmg (486 total enr+xrr) is maxing at 325 on a 50/50.

Full relog does not fix.

**Comment by NCKryptonite** [comment]
Also, AoE dmg seems to be blocked by corpses? I'll often fire into a crowd of 5+ & only 2 get hit. Not a display error, since I'll fire directly on one on the lower ledge in el farid, see damage numbers from mobs on the ground, and the one I fired at is still at full health after 5 shots.

**Comment by ghost** [comment]
I've noticed similar behavior with the PSI multi energy bolt not dealing damage to the mob I hit/aiming at.

**Comment by Corruptforce** [comment]
I think we need to re-evaluate after the balance patch.
Quite often buying higher TL weapons is not a damage increase (mainly I suspect due to low % percentage due to requiring higher weaponskill (although doesn't seem to be the case for this one)).

I think the flow would be better for leveling and true newbies if higher TL weapon had higher base damage than the last lower TL iteration (might need to work on % scaling so when you hit max dmg% that it's not way too high).

Sometimes it's a bummer getting a new psi spell version and realising the TTK is much higher (mainly due to freq and damage being way lower).

**Comment by cptYokom** [comment]
Did you take into account the range to target, all raygun weps do less damage at longer range. A small change is target range can make a big change to damage. Like the all the fusion weps do more at longer range.

**Comment by NCKryptonite** [comment]
@cptYokom Yes. Even though we're talking about being in el farid. So damage is maybe 5 less from not standing over the scorpion.

**Comment by cptYokom** [comment]
Noticed on this issue at the HEW in p2 there is two tl90 rayguns. One is NCPD and the other is the DOY both say special and have what looks like the same stats.

**Comment by ghost** [comment]
@cptYokom The issue with HEW selling a NCPD and DOY RayGun Cannon have already been delt with for the next patch.
